foo.js

```diff
@@ -0,0 +1,18 @@
+import pl from "npm:nodejs-polars";
```

Comment:
Is this and `foods.csv` supposed to be on the root of the repository?

Reply:
Thanks, they were left by mistake